name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/SantanderConsumerBankPDFExtractor.java

```diff
@@ -185,4 +193,69 @@ private <T extends Transaction<?>> void addFeesSectionsTransaction(T transaction
                 .match("^Fremde Abwicklungsgeb.hr .* (?<fee>[.,\\d]+)\\- (?<currency>[\\w]{3})$")
                 .assign((t, v) -> processFeeEntries(t, v, type));
     }
+
+    private void addKontoauszugDocument()
```

Comment:
We should stick to a standard here, which is also stored in the [Contributing rules](https://github.com/Nirus2000/portfolio/blob/master/CONTRIBUTING.md#auxiliary-classes). It is again a variant for one and the same thing, the reading of account or deposit statements.
Again and again other variants, means always more maintenance effort.
Please change to the variant of the DKB Importer with the defined variables and descriptions.

Reply:
Can you please have a look now? I think now it should fit